man/sd_id128_to_string.xml

```diff
@@ -63,7 +63,7 @@
     easier to use a format string for
     <citerefentry project='man-pages'><refentrytitle>printf</refentrytitle><manvolnum>3</manvolnum></citerefentry>.
     This is easily done using the
-    <function>SD_ID128_FORMAT_STR</function> and
+    <function>SD_ID128_FORMAT_STR()</function> and
```

Comment:
Actually, SD_ID128_FORMAT_STR does not take any parameters:
```C
src/systemd/sd-id128.h:#define SD_ID128_FORMAT_STR "%02x%02x%02x%02x%02x%02x%02x%02x%02x%02x%02x%02x%02x%02x%02x%02x"
```
So the change needs to be in the other direction: strip () after all instances.